nvdaHelper/vbufBase/storage.cpp

```diff
@@ -410,9 +410,13 @@ std::wstring VBufStorage_textFieldNode_t::getDebugInfo() const {
 void VBufStorage_buffer_t::forgetControlFieldNode(VBufStorage_controlFieldNode_t* node) {
 	nhAssert(node); //Node can't be NULL
 	map<VBufStorage_controlFieldNodeIdentifier_t,VBufStorage_controlFieldNode_t*>::iterator i=controlFieldNodesByIdentifier.find(node->identifier);
-	nhAssert(i!=controlFieldNodesByIdentifier.end());
-	nhAssert(i->second==node);
-	controlFieldNodesByIdentifier.erase(i);
+	if(i==controlFieldNodesByIdentifier.end()) {
+		// Node not known
+		return;
+	}
+	if(i->second==node) {

```

Comment:
These two asserts have been converted into if checks. Is there any reason these cases should now be happening in normal operation where they didn't before? If not, this change should be removed.